src/go/k8s/pkg/resources/configuration/configuration.go

```diff
@@ -0,0 +1,154 @@
+// Package configuration provide tools for the operator to manage cluster configuration
+package configuration
+
+import (
+	"crypto/md5" // nolint:gosec // this is not encrypting secure info
+	"fmt"
+	"reflect"
+	"strings"
+
+	"github.com/vectorizedio/redpanda/src/go/k8s/pkg/resources/featuregates"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/api/admin"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/config"
+)
+
+const (
+	// useMixedConfiguration can be temporarily used until .boostrap.yaml is fully supported.
+	useMixedConfiguration = true
+
+	redpandaPropertyPrefix = "redpanda."
+)
+
+// knownNodeProperties is initialized with all well known node properties from the "redpanda" configuration tree.
+var knownNodeProperties map[string]bool
+
+// otherKnownNodeProperties contains node properties that are not listed in the struct
+var otherKnownNodeProperties = map[string]bool{
+	"redpanda.enable_central_config": true, // deprecated but may be needed in some tests
```

Comment:
This is no longer needed with latest redpanda, all references to `enable_central_config` can be removed.